app/models/service_terraform_template.rb

```diff
@@ -97,6 +97,17 @@ def job_option_key(action)
     "#{action.downcase}_job_options".to_sym
   end
 
+  def parse_dialog_options
+    dialog_options = options[:dialog] || {}
+
+    params = dialog_options.each_with_object({}) do |(attr, val), obj|
+      var_key = attr.sub(/^(password::)?dialog_/, '')
```

Comment:
are the password values encrypted?  If so, do they need to be decrypted?

Reply:
They can be, and yes they do need to be